introduction.tex

```diff
@@ -84,7 +84,8 @@ \subsection{Register Definition Format}
         \hline
         W & Write-only. When read this field returns 0. \\
         \hline
-        W1 & Write-only. Only writing 1 has an effect. \\
+        W1 & Write-only. Every time that 1 is written an action is taken. When
```

Comment:
I will need to go back through and see if this is correct. Is it really "every time"? I don't think that's true.

Reply:
Do you mean "An action may be taken only when a 1 is written. Writing 0 has no effect."

Reply:
That doesn't feel strong enough.

I want every write of 1, regardless of the previously written value, to take the action. Eg. every time haltreq is written, we request that the hart halts. (This is ignored if the hart is already halted.) Likewise, every time setresethaltreq is written high, it sets the conceptual resethaltreq bit high. If it's already high, that's a no-op, but the attempt is made.

I believe this was a possible interpretation of the previous language, and I want to be clear that it's the only interpretation.

Does that make sense?

Reply:
haltreq is actually not a W1 bit. Pretend I wrote an example with dmireset.

Reply:
I guess I dont' see that as more clear than the current "has an effect".

Reply:
OK. I've reverted this definition.